script.lua

```diff
@@ -2020,7 +2020,7 @@ function MajorPowerC(obj, player_color, alt_click)
     if alt_click then
         cards = 2
     end
-    startDealPowerCards(false, Player[player_color], cards)
+    startDealPowerCards({minor = false, player = Player[player_color], count = cards})
```

Comment:
matching the ordering of `modifyCardGain` the boolean parameter should be the middle argument